<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTDClass</class>
 <widget class="QMainWindow" name="GTDClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GTD</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 125, 125);
border: 5px solid rgb(95,95, 95);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="Tabla">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>801</width>
       <height>521</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
Border:0px;</string>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lentillas</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Observaciones</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Venta</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expiracion</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Restante</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Nuevo">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 3px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy: -0.3,
radius: 1.35, stop: 0 rgba(21, 230,226,255), stop: 1 rgba(21, 230,226, 100));

min-width: 70px;
}

QPushButton:hover {
     background-color: rgb(21, 230, 255);

 }

QPushButton:pressed{
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy:-0.3,
radius: 1.3, stop: 0 rgba(21,230,226, 100), stop: 1 rgba(21, 230,226,255));
}

</string>
     </property>
     <property name="text">
      <string>Nuevo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Eliminar">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: white;
border: 3px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy: -0.3,
radius: 1.35, stop: 0 rgba(21, 230,226,255), stop: 1 rgba(21, 230,226, 100));

min-width: 70px;
}

QPushButton:hover {
     background-color: rgb(21, 230, 255);

 }

QPushButton:pressed{
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy:-0.3,
radius: 1.3, stop: 0 rgba(21,230,226, 100), stop: 1 rgba(21, 230,226,255));
}

</string>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QFrame" name="Formulario">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(21, 180, 200);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nombre">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>15</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border: 2px solid grey;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Pack">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid grey;</string>
      </property>
      <item>
       <property name="text">
        <string>1 mes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11 meses</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12 meses</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Aceptar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: white;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy: -0.3,
radius: 1.35, stop: 0 rgba(21, 230,226,255), stop: 1 rgba(21, 230,226, 100));

min-width: 70px;
}

QPushButton:hover {
     background-color: rgb(21, 230, 255);

 }




QPushButton:pressed{
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy:-0.3,
radius: 1.3, stop: 0 rgba(21,230,226, 100), stop: 1 rgba(21, 230,226,255));
}


</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cancelar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: white;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy: -0.3,
radius: 1.35, stop: 0 rgba(21, 230,226,255), stop: 1 rgba(21, 230,226, 100));

min-width: 70px;
}

QPushButton:hover {
     background-color: rgb(21, 230, 255);

 }




QPushButton:pressed{
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy:-0.3,
radius: 1.3, stop: 0 rgba(21,230,226, 100), stop: 1 rgba(21, 230,226,255));
}


</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Tipo lentillas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Pack:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Lentillas">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border: 2px solid grey;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>225</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Fecha entregada:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Fecha_Entrada">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: black;
border: 2px solid #555;
border-radius: 0px;
padding: 5px;
background: qradialgradient(cx: 0.4, cy: -0.3,
fx: 0.4, fy: -0.3,
radius: 1.35, stop: 0 rgba(255, 255,255,255), stop: 1 rgba(255, 255,255, 255));

min-width: 70px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>95</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Observaciones:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Observaciones">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border: 2px solid grey;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="Busqueda">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>190</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border: 2px solid grey;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border: 0px;</string>
     </property>
     <property name="text">
      <string>Busqueda:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Lista_Busqueda">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>220</y>
       <width>221</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F_Calendario">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCalendarWidget" name="Calendario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>256</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gtd.qrc"/>
 </resources>
 <connections/>
</ui>
